mne/time_frequency/tests/test_psd.py

```diff
@@ -127,10 +135,10 @@ def test_compares_psd():
     n_fft = 2048
 
     # Compute psds with the new implementation using Welch
-    psds_welch, freqs_welch = compute_raw_psd(raw, tmin=tmin, tmax=tmax,
-                                              fmin=fmin, fmax=fmax,
-                                              proj=False, picks=picks,
-                                              n_fft=n_fft, n_jobs=1)
+    psds_welch, freqs_welch = psd_welch(raw, tmin=tmin, tmax=tmax,
+                                        fmin=fmin, fmax=fmax,
+                                        proj=False, picks=picks,
+                                        n_fft=n_fft, n_jobs=1)
```

Comment:
I added it for the test_psd function. what would be the benefit for adding it here? It's just testing that the output of the Welch psd is the same as the output of MPL psd. I could shorten the tmin/tmax to make it faster if that's the main benefit.